src/containers/Bounty/components/modals/FulfillBountyFormModal.js

```diff
@@ -123,7 +123,7 @@ let FulfillBountyFormModalComponent = props => {
           >
             Cancel
           </Button>
-          <Button disabled={uploading} type="primary" buttonType="button">
+          <Button disabled={uploading} type="primary" buttonType="submit">
```

Comment:
@c-o-l-o-r this is just html default behavior. buttonType="button" will not trigger a form submit.

Reply:
@villanuevawill sure, but why does it in all the other modals?